docs/api.rst

```diff
@@ -10740,3 +10740,220 @@ Importing a database snapshot
+
+    **Example Request**
+
+    .. http:example:: curl wget httpie python-requests
+
+        PATCH /api/1/namings/addressbook/private HTTP/1.1
+        Host: localhost:5042
+        Content-Type: application/json;charset=UTF-8
+
+        {
+          "entries": [
+            {"address": "0x9531c059098e3d194ff87febb587ab07b30b1306", "name": "Dude ABC"},
+            {"address": "0x8A4973ABBCEd48596D6D79ac6B53Ceda65e342CD", "name": "Dude XYZ"}
+          ]
+        }
+
+    :reqjson object entries: A list of entries to be updated in the addressbook
+
+    **Example Response**
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Content-Type: application/zip
+
+        {
+            "result": True,
+            "message": ""
+        }
+
+    :resjson bool result: A boolean which is true in case if entries were updated successfully.
+    :resjson str message: Error message if any errors occurred.
+    :statuscode 200: Entries were updated successfully.
+    :statuscode 400: Provided JSON is in some way malformed.
+    :statuscode 409: Some of the provided entries don't exist in the addressbook or no user is currently logged in or addresses have incorrect format.
+    :statuscode 500: Internal rotki error.
+
+
+Delete mappings in the addressbook
+==================================
+
+.. http:delete:: /api/(version)/namings/addressbook
+
+    Doing a DELETE on addressbook endpoint with either /global or /private postfix with a list of addresses will delete mappings in the addressbook of the specified addresses
+
+    **Example Request**
+
+    .. http:example:: curl wget httpie python-requests
+
+        DELETE /api/1/namings/addressbook/global HTTP/1.1
+        Host: localhost:5042
+        Content-Type: application/json;charset=UTF-8
+
+        {
+          "addresses": ["0x9531c059098e3d194ff87febb587ab07b30b1306", "0x8A4973ABBCEd48596D6D79ac6B53Ceda65e342CD"]
+        }
+
+    :reqjson object entries: A list of addresses to be deleted from the addressbook
+
+    **Example Response**
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Content-Type: application/zip
+
+        {
+            "result": True,
+            "message": ""
+        }
+
+    :resjson bool result: A boolean which is true in case if entries were deleted successfully.
+    :resjson str message: Error message if any errors occurred.
+    :statuscode 200: Entries were deleted successfully.
+    :statuscode 400: Provided JSON is in some way malformed.
+    :statuscode 409: Some of the provided entries don't exist in the addressbook or no user is currently logged in or addresses have incorrect format.
+    :statuscode 500: Internal rotki error.
+
+
+Search for all known names of an address
+========================================
+
+.. http:post:: /api/(version)/namings
```

Comment:
Not set on it. But mostly curious. Why not use names/search and just use names/ here?